catalyst_voices/packages/catalyst_voices_brands/lib/src/themes/extended_color_scheme.dart

```diff
@@ -0,0 +1,219 @@
+import 'package:flutter/material.dart';
+
+@immutable
+class ExtendedColorScheme extends ThemeExtension<ExtendedColorScheme> {
```

Comment:
How about: CatalystColorScheme?

Reply:
I was using something like this at the beginning but I changed it because I thought we need a more generic name that can be used also for other brands.
Maybe `VoicesColorScheme`? - so that it's related to the platform and not to the brand?

Reply:
Aren't those the same thing? Catalyst & Voices? Imho this name is also good

Reply:
We explicitly use `Catalyst` as a brand name so I think it could be misleading to use it here. I opted for `VoicesColorScheme`.